keyboards/hexboard/1_1/info.json

```diff
@@ -0,0 +1,446 @@
+            {"matrix": [9, 5], "x": 74, "y": 39, "flags": 8},
+            {"matrix": [9, 6], "x": 74, "y": 45, "flags": 8},
+            {"matrix": [9, 7], "x": 74, "y": 51, "flags": 8},
+            {"matrix": [9, 8], "x": 74, "y": 57, "flags": 8},
+            {"matrix": [9, 9], "x": 74, "y": 64, "flags": 8},
+            {"matrix": [10, 0], "x": 37, "y": 0, "flags": 8},
+            {"matrix": [10, 1], "x": 56, "y": 12, "flags": 8},
+            {"matrix": [10, 2], "x": 56, "y": 18, "flags": 8},
+            {"matrix": [10, 3], "x": 56, "y": 24, "flags": 8},
+            {"matrix": [10, 4], "x": 56, "y": 30, "flags": 8},
+            {"matrix": [10, 5], "x": 56, "y": 36, "flags": 8},
+            {"matrix": [10, 6], "x": 56, "y": 42, "flags": 8},
+            {"matrix": [10, 7], "x": 56, "y": 48, "flags": 8},
+            {"matrix": [10, 8], "x": 56, "y": 54, "flags": 8},
+            {"matrix": [10, 9], "x": 56, "y": 60, "flags": 8},
+            {"matrix": [11, 0], "x": 37, "y": 9, "flags": 8},
+            {"matrix": [11, 1], "x": 37, "y": 15, "flags": 8},
+            {"matrix": [11, 2], "x": 37, "y": 21, "flags": 8},
+            {"matrix": [11, 3], "x": 37, "y": 27, "flags": 8},
+            {"matrix": [11, 4], "x": 37, "y": 33, "flags": 8},
+            {"matrix": [11, 5], "x": 37, "y": 39, "flags": 8},
+            {"matrix": [11, 6], "x": 37, "y": 45, "flags": 8},
+            {"matrix": [11, 7], "x": 37, "y": 51, "flags": 8},
+            {"matrix": [11, 8], "x": 37, "y": 57, "flags": 8},
+            {"matrix": [11, 9], "x": 37, "y": 64, "flags": 8},
+            {"matrix": [12, 0], "x": 18, "y": 3, "flags": 8},
+            {"matrix": [12, 1], "x": 18, "y": 12, "flags": 8},
+            {"matrix": [12, 2], "x": 18, "y": 18, "flags": 8},
+            {"matrix": [12, 3], "x": 18, "y": 24, "flags": 8},
+            {"matrix": [12, 4], "x": 18, "y": 30, "flags": 8},
+            {"matrix": [12, 5], "x": 18, "y": 36, "flags": 8},
+            {"matrix": [12, 6], "x": 18, "y": 42, "flags": 8},
+            {"matrix": [12, 7], "x": 18, "y": 48, "flags": 8},
+            {"matrix": [12, 8], "x": 18, "y": 54, "flags": 8},
+            {"matrix": [12, 9], "x": 18, "y": 60, "flags": 8},
+            {"matrix": [13, 0], "x": 0, "y": 9, "flags": 8},
+            {"matrix": [13, 1], "x": 0, "y": 15, "flags": 8},
+            {"matrix": [13, 2], "x": 0, "y": 21, "flags": 8},
+            {"matrix": [13, 3], "x": 0, "y": 27, "flags": 8},
+            {"matrix": [13, 4], "x": 0, "y": 33, "flags": 8},
+            {"matrix": [13, 5], "x": 0, "y": 39, "flags": 8},
+            {"matrix": [13, 6], "x": 0, "y": 45, "flags": 8},
+            {"matrix": [13, 7], "x": 0, "y": 51, "flags": 8},
+            {"matrix": [13, 8], "x": 0, "y": 57, "flags": 8},
+            {"matrix": [13, 9], "x": 0, "y": 64, "flags": 8}
+        ]
+    },
+    "usb": {
+        "device_version": "1.1.0",
+        "pid": "0xCAFE",
+        "vid": "0x239B"
```

Comment:
This vendor ID is in use, and should be changed.   The further past 0x4000, the better,

https://www.the-sz.com/products/usbid/index.php?v=0x239B&p=&n=

Reply:
I picked 575C for the vendor, and 6B01, 6B02 for the product